commands/displayers/droplet_supported_backup_policies.go

```diff
@@ -0,0 +1,44 @@
+package displayers
+
+import (
+	"io"
+
+	"github.com/digitalocean/doctl/do"
+)
+
+type DropletSupportedBackupPolicy struct {
+	DropletSupportedBackupPolicies []do.DropletSupportedBackupPolicy
+}
+
+var _ Displayable = &DropletSupportedBackupPolicy{}
+
+func (d *DropletSupportedBackupPolicy) JSON(out io.Writer) error {
+	return writeJSON(d.DropletSupportedBackupPolicies, out)
+}
+
+func (d *DropletSupportedBackupPolicy) Cols() []string {
+	cols := []string{
+		"Name", "PossibleWindowStarts", "WindowLengthHours", "RetentionPeriodDays", "PossibleDays",
+	}
+	return cols
+}
+
+func (d *DropletSupportedBackupPolicy) ColMap() map[string]string {
+	return map[string]string{
+		"Name": "Name", "PossibleWindowStarts": "Possible Window Starts",
+		"WindowLengthHours": "Window Length Hours", "RetentionPeriodDays": "Retention Period Days", "PossibleDays": "Possible Days",
+	}
+}
+
+func (d *DropletSupportedBackupPolicy) KV() []map[string]any {
+	out := make([]map[string]any, 0)
+	for _, d := range d.DropletSupportedBackupPolicies {
```

Comment:
Same thing here:

```suggestion
func (d *DropletSupportedBackupPolicy) KV() []map[string]any {
	out := make([]map[string]any, 0)
	for _, supported := range d.DropletSupportedBackupPolicies {
```

Reply:
Done in [bc4ede2](https://github.com/digitalocean/doctl/pull/1609/commits/bc4ede23d4ca84d1ce6e2cd3361c0cbb095f24c7)